packages/web/src/components/collections-table/CollectionsTable.tsx

```diff
@@ -137,7 +137,7 @@ export const CollectionsTable = ({
   const overflowMenuRef = useRef<HTMLDivElement>(null)
   const renderOverflowMenuCell = useCallback((cellInfo: CollectionCell) => {
     const collection = cellInfo.row.original
-    const Icon = collection.is_private ? IconVisibilityHidden : IconLock
+    const Icon = collection.is_private ? IconVisibilityHidden : IconCart
```

Comment:
is there no case where the collection is public?

Reply:
there is, but then we check for `shouldShowIcon` after.. however i think the pattern i used in TracksTable maybe is better..